sdk/go/google/accesscontextmanager/v1/getAccessLevel.go

```diff
@@ -0,0 +1,37 @@
+// *** WARNING: this file was generated by the Pulumi SDK Generator. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+package v1
+
+import (
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+// Get an Access Level by resource name.
+func LookupAccessLevel(ctx *pulumi.Context, args *LookupAccessLevelArgs, opts ...pulumi.InvokeOption) (*LookupAccessLevelResult, error) {
```

Comment:
Why are the Go functions named differently than the other SDKs (Lookup prefix vs Get prefix)?

Reply:
I don't know but this seems to be universal:

- Azure-Native: https://github.com/pulumi/pulumi-azure-native/blob/master/sdk/go/azure/cdn/getEndpoint.go
- GCP: https://github.com/pulumi/pulumi-gcp/blob/master/sdk/go/gcp/compute/getInstance.go

Reply:
Its another go codegen thing: https://github.com/pulumi/pulumi/blob/master/pkg/codegen/go/gen.go#L2061. Not entirely sure why though.